docker/Settings.example.yml

```
@@ -1,59 +1,60 @@
-    ImagesUntilDate: '2020-01-02'
-    Rating: null
-    Albums:
-      - UUID
-    ExcludedAlbums:
-      - UUID
-    People:
-      - UUID
-    Tags:
-      - Vacation
-      - Travel/Europe
+General:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)